ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4069,6 +4069,22 @@ AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC2X_VOADM2X_SAIDC2X_RAIDC2X_SSPC:
   - RAIDC2X
   - SSPC
 
+
+AHU_SS_RMM_RTM_FDPM_EFPC_SFPC_HWSC_ESDC_FSAM:
+  description: "AHU with supply and exhaust pressure, flowrate control, cooling and heating valve control, fire and smoke alarms"
```

Comment:
You note that this unit has cooling, but there are not any cooling ABSTRACT types applied.